phoenix-core/src/main/java/org/apache/phoenix/util/ViewUtil.java

```diff
@@ -954,7 +954,9 @@ public static void addTagsToPutsForViewAlteredProperties(List<Mutation> tableMet
             PTable parent, ExtendedCellBuilder extendedCellBuilder) {
         byte[] parentUpdateCacheFreqBytes = null;
         byte[] parentUseStatsForParallelizationBytes = null;
-        byte[] parentPhoenixTTLBytes = null;
+        //Commenting out phoenixTTL property to exclude.
+        //TODO:- re-enable after introducing TTL for views.
+        //byte[] parentPhoenixTTLBytes = null;
```

Comment:
does this mean we need to re-enable all the commented out code in this class?

Reply:
Yes, if we want the prop out of sync with table which I think we will. I commented out so that in next change it won't be missed. I can remove it as well!